packages/core/lib/debug/interpreter.js

```diff
@@ -761,6 +778,21 @@ class DebugInterpreter {
     }
     return tempPrintouts;
   }
+
+  // update the numbers of lines to be printed from options -<num>|+<num> from user args
+  updatePrintoutLines(userArgs, currentLines) {
+    let beforeLines = currentLines[0], afterLines = currentLines[1];
```

Comment:
Note that you can use destructuring to write this as just `let [beforeLines, afterLines] = currentLines;`.  (Of course, really you should be using objects rather than arrays, but you can destructure those too. :) )